src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -3,18 +3,9 @@
 from dataclasses import dataclass
 
 from databricks.sdk import WorkspaceClient
-from databricks.sdk.errors import NotFound
-from databricks.sdk.service.compute import ClusterDetails, ClusterSource, Policy
+from databricks.sdk.service.compute import ClusterSource
 
-from databricks.labs.ucx.assessment.crawlers import (
-    _AZURE_SP_CONF_FAILURE_MSG,
-    INCOMPATIBLE_SPARK_CONFIG_KEYS,
-    _azure_sp_conf_in_init_scripts,
-    _azure_sp_conf_present_check,
-    _get_init_script_data,
-    logger,
-    spark_version_compatibility,
-)
+from databricks.labs.ucx.assessment.crawlers import _check_cluster_failures, logger
```

Comment:
don't import logger, initialize one in the top of the module.

Reply:
fixed